cmd/car/create.go

```diff
@@ -101,6 +105,19 @@ func writeFiles(ctx context.Context, bs *blockstore.ReadWrite, paths ...string)
 		}, nil
 	}
 
+	if noWrap {
+		root, _, err := builder.BuildUnixFSRecursive(paths[0], &ls)
+		if err != nil {
+			return cid.Undef, err
+		}
+		rcl, ok := root.(cidlink.Link)
+		if !ok {
+			return cid.Undef, fmt.Errorf("could not interpret %s", root)
+		}
```

Comment:
is there a reason to do this as a separate block here, versus taking the `l`on line 123, setting it as `root` if the flag is set and breaking out of the for loop?

Reply:
Not really, I can change that